x/params/keeper/migrations.go

```diff
@@ -0,0 +1,24 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	v046 "github.com/cosmos/cosmos-sdk/x/params/migrations/v046"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper Keeper
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(keeper Keeper) Migrator {
+	return Migrator{
+		keeper: keeper,
+	}
+}
+
+// Migrate2to3 migrates x/staking state from consensus version 2 to 3.
```

Comment:
```suggestion
// Migrate1to2 migrates baseapp from consensus version 1 to 2.
```

Do you think this belongs here? Maybe we need a separate package for baseapp? This doesn't seem right.